src/app/newcomers/newcomers.component.scss

```diff
@@ -1,59 +1,63 @@
 .colour1{
-    background-color: #8d2165;
+  background-color: #8d2165;
 }
 .fa-long-arrow-down
 {
-    font-size:48px;
+  font-size:48px;
 }
 
 .modal {
-  text-align: center;
+text-align: center;
```

Comment:
It would be great if this code was consistently formatted with spacing - is it 2 spaces?

ex - easier to read if it has 2 spaces like this: 
```
.modal
{
    text-align: center;
}
```

Reply:
Okay sure. Will reindent the code :)